### ISSUE TITLE
Assigned jobs do not show the person that was assigned until another add command is input

### ISSUE THREAD

**Issue by Btaykb** [Btaykb/pe#5]
After assigning a job to a person, the UI does not immediately show that the person has been assigned the job. 

After input `assign 2 i/7`
![image.png](https://raw.githubusercontent.com/Btaykb/pe/main/files/fc6b7c92-bc9a-4c5e-8eb5-e8b920d0a427.png)

Before input command `add n/clean d/8`, still not showing `nicole` as being assigned to job 2
![image.png](https://raw.githubusercontent.com/Btaykb/pe/main/files/4aa376da-776b-4bf6-85ed-c70c5aca50e8.png)

Only after executing the `add` command does `nicole` appear as the assigned employee
![image.png](https://raw.githubusercontent.com/Btaykb/pe/main/files/71ee52c9-dc61-4620-9f25-6356e0eeb52d.png)

**Comment by nus-pe-script** [comment]
# Team's Response
No details provided by team.

## The 'Original' Bug

[The team marked this bug as a duplicate of the following bug]

># Job display list does not update to show assigned persons after executing assign command
>
>Steps to reproduce
>1. Execute any valid assign command
>
>Expected: Job display list automatically updates to show the persons assigned to the job specified by the executed assign command
>
>Actual: Job display list is not updated
>
>
>![2022-04-16_14-52.png](https://raw.githubusercontent.com/nus-cs2103-AY2122S2/pe/master/files/400c4a2b-8924-4048-87fb-f87c848f3005.png)
>
>In this screenshot, the command `assign 1 i/1` was successfully executed, but the "Assigned" cell in the job display list was not updated.
>
>Put severity as high since this is a very common scenario that can cause major inconvenience to users. The only way to get the list to update is to perform a `find`, `add`, `delete`, or `mark` command as far as I could tell. 
>
>
>
>
><hr>
><sub>[original: nus-cs2103-AY2122S2/pe-interim#2962] [original labels: type.FunctionalityBug severity.High]</sub>
>
>
>

## Their Response to the 'Original' Bug

[This is the team's response to the above 'original' bug]

>We agree that most users will face the bug, as assigning jobs will be done frequently.
>
>Downgraded to `Medium` because it does not make the app almost unusable, and there are no big side effects such as data loss. It is just the UI not updating. The new assignments will be displayed after performing the other commands as mentioned by you. i.e. the list will update after performing a `find`, `add`, `delete`, or `mark` command.
>
>This bug only appears when the user assigns the command, and the user can still use the app afterwards. Hence, the app will not be considered almost unusable.
>
>### According to the PE instructions
>> severity.High : A flaw that affects most users and causes major problems for users. i.e., only problems that make the product **almost unusable** for most users should have this label.


# Items for the Tester to Verify
## :question: Issue duplicate status

Team chose to mark this issue as a duplicate of another issue (as explained in the _**Team's response**_ above)



- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

-------------------
## :question: Issue severity

Team chose [`severity.Medium`]
Originally [`severity.VeryLow`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

-------------------
